muxpi/set-up-image.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -e
+
+#If you want to run the neptune image, run `./set-up-image.sh neptune`
+NEPTUNE_IMAGE="$1"
+
+HOME="/home/muxpi"
+IMAGES="${HOME}/images"
+
+#The url for the latest successful nightly build.
+URL_MINIMAL="https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY/lastSuccessfulBuild/artifact/artifacts_rpi/core-image-pelux-minimal-dev-raspberrypi3.rpi-sdimg"
+VARIANT_MINIMAL="core-image-pelux-minimal-dev-raspberrypi3.rpi-sdimg"
+
+URL_NEPTUNE="https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY/lastSuccessfulBuild/artifact/artifacts_rpi-qtauto/core-image-pelux-qtauto-neptune-dev-raspberrypi3.rpi-sdimg"
+VARIANT_NEPTUNE="core-image-pelux-qtauto-neptune-dev-raspberrypi3.rpi-sdimg"
+
+VARIANT=""
+URL=""
+if [ "$NEPTUNE_IMAGE" == neptune ]; then
+   VARIANT=$VARIANT_NEPTUNE
+   URL=$URL_NEPTUNE
+else 
+   VARIANT=$VARIANT_NEPTUNE
+   URL=$URL_MINIMAL
+fi
+
+mkdir -p $IMAGES
+
+wget $URL -P $IMAGES
+if [ $? == 0 ]; then
+   echo "Image downloaded"
+fi
+echo "{\"${VARIANT}\":\"\"}" > $HOME/map.json
+echo "Json map is ready. Compressing the downloaded image... "
+   
+cd $IMAGES
+tar -czvf $VARIANT.tar.gz $VARIANT
```

Comment:
I guess this is the simplest solution. But could also send a patch to the fota (?) tool so we don't have to waste cycles on gzip:ing.

Reply:
Future work